recipes/faampy/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "faampy" %}
+{% set version = "0.1.2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/ncasuk/faampy.git
+    git_tag: {{ version }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - python >=2.7,<=3.5
```

Comment:
Use the skip statement in the build section above and remove the Python constraint here and below. I guess that would be:

```yaml
build:
    number: 0
    skip:  True  # [py>36]
```